ads/popin.md

```diff
@@ -26,7 +26,7 @@ limitations under the License.
     height=568
     layout=responsive
     heights="(min-width:1907px) 39%, (min-width:1200px) 46%, (min-width:780px) 64%, (min-width:480px) 98%, (min-width:460px) 167%, 196%"
-    data-mediaid="popin_amp"
+    data-mediaid="popin_amp">
```

Comment:
Oh it's just styling. I mean
```
 <amp-ad width="300" height="568"
      type="popin"
      layout=responsive
      heights="(min-width:1907px) 39%, (min-width:1200px) 46%, (min-width:780px) 64%, (min-width:480px) 98%, (min-width:460px) 167%, 196%"
      data-mediaid="popin_amp">
  </amp-ad>
```
(copied from the example provided in `ads.amp.html`)